File: examples-sdk/swift/039-basic-observer-dashboard/main.swift
// 039-basic-observer-dashboard — Swift SDK equivalent
// Compile: vil compile --from swift --input 039-basic-observer-dashboard/main.swift --release

let server = VilServer(name: "observer-demo", port: 8080)
let demo = ServiceProcess(name: "demo")
demo.endpoint(method: "GET", path: "/hello", handler: "hello")
demo.endpoint(method: "POST", path: "/echo", handler: "echo")
server.service(demo)
server.compile()

File: examples-sdk/csharp/039-basic-observer-dashboard/Main.cs
// 039-basic-observer-dashboard — C# SDK equivalent
// Compile: vil compile --from csharp --input 039-basic-observer-dashboard/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("observer-demo", 8080);
var demo = new ServiceProcess("demo");
demo.Endpoint("GET", "/hello", "hello");
demo.Endpoint("POST", "/echo", "echo");
server.Service(demo);
server.Compile();
